lambify/docker-compose.yml

```diff
@@ -4,10 +4,15 @@ services:
     build:
       context: '.'
       dockerfile: Dockerfile-build
+    entrypoint: ./bin/entrypoint
```

Comment:
Thanks to this docker compose feature request for the idea. https://github.com/docker/compose/issues/6736